functests/10-test-template-validator-placement.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+SCRIPTPATH=$( dirname $(readlink -f $0) )
+source ${SCRIPTPATH}/testlib.sh
+
+RET=0
+TEST_NS="${KV_NAMESPACE}"
+
+oc create -n ${TEST_NS} -f "${SCRIPTPATH}/10-template-validator-unversioned-cr.yaml" || exit 2
+
+# Wait for the operator to create the deployment, we don't care if pods are actually ready, as
+# we only check for the pod scheduling fields
+DEPLOYMENT_FOUND=false
+for i in {1..20}; do
+  oc get -n ${TEST_NS} deploy virt-template-validator
```

Comment:
Do we want to see the `oc get` output, or redirect it to `/dev/null`?
I think we should redirect, but it may be useful for debugging.

Reply:
when running `make functests` the actual test files are wrapped with `test-runner.sh` which encapsulates `stdout` unless a test fails, see here: https://github.com/kubevirt/kubevirt-ssp-operator/blob/master/functests/test-runner.sh#L37

Reply:
Ok, I didn't know about that.